ci/deb-ppa-upload/local-run.sh

```diff
@@ -4,7 +4,7 @@ set -e -o pipefail -u
 
 source ../local-run-commons.sh deb-ppa-upload
 check_env gpg-ssh.env
-export_hash deb-ppa-upload
```

Comment:
Pls try running all 3 scripts locally to verify! (unless you've done so already)